.travis.yml

```diff
@@ -27,8 +27,8 @@ cache:
 env:
   matrix:
     # Test builds first, since they don't upload things.
-    - TEST_GOAL=coverage
-    #- TEST_GOAL=performance
+    - TEST_GOAL=coverage TOXCORE=toktok/c-toxcore
+    - TEST_GOAL=performance TOXCORE=irungentoo/toxcore
```

Comment:
Right, I want a "performance" test goal, but that goal is testing performance of tox4j, not toxcore, so I can also use it to check whether it compiles with iru/toxcore at the same time.